packages/services/src/services/VideoPlayer/VideoPlayer.js

```diff
@@ -0,0 +1,190 @@
+/**
+ * These id's for production use of IBM.com
+ * @type {number} _partnerId The ID of your Kaltura account (aka partnerId)
+ * @type {number} _uiConfId The ID of the Kaltura player to use
+ * @private
+ */
+const _partnerId = process.env.KALTURA_PARTNER_ID || 243342;
+const _uiConfId = process.env.KALTURA_UICONF_ID || 12905712;
+
+/**
+ * @type {string} _embedUrl The API URL to call
+ * @private
+ */
+const _embedUrl = `https://cdnapisec.kaltura.com/p/${_partnerId}/sp/${_partnerId}00/embedIframeJs/uiconf_id/${_uiConfId}/partner_id/${_partnerId}`;
+
+/**
+ *
+ * Number of times to retry the script ready loop before failing
+ * @type {number}
+ * @private
+ */
+
+const _timeoutRetries = 50;
+/**
+ *
+ * Tracks the number of attempts for the script ready loop
+ * @type {number}
+ * @private
+ */
+let _attempt = 0;
+
+/**
+ *
+ * Tracks the script either in loaded state or loading state
+ * @type {boolean} _scriptLoaded to track the script loaded or not
+ * * @type {boolean} _scriptLoading to track the script loading or not
+ * @private
+ */
+let _scriptLoaded = false;
+let _scriptLoading = false;
```

Comment:
@mkothur can you add two separate JSDoc comments for `_scriptLoaded` and `_scriptLoading`?

Reply:
done!